projects.json

```diff
@@ -1562,7 +1586,7 @@
     "isLive": true,
     "name": "MEXC",
     "url": "https://www.mexc.com/",
-    "isExternalExchange": true,
+    "showAccountTag": true,
```

Comment:
can we keep both "isExternalExchange" and "showAccountTag" just to make sure we dont break current explorer?

Reply:
by "break" I mean the account tag keeps showing up even after we merge this PR, so we dont need coordinated releases